src/components/UnitToggle.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ToggleSwitch from '@trendmicro/react-toggle-switch';
+
+const UnitToggle = props => {
+  const { unitChange, checkUnit } = props;
```

Comment:
can you destructure the props on the arguments

Reply:
I'm confused on what else needs destructuring here?